### ISSUE TITLE
feature request; hashrate history graph.

### ISSUE THREAD

**Issue by Postur** [UnamSanctam/UnamWebPanel#107]
# feature request; Hashrate history graph.

pretty self explanatory. 
I don't know if the data is already logged, but having some kind of historic graph of hashrate would be very nice.

I could help with implementing, but php is not really my cup of tea.

- Thank you

**Comment by UnamSanctam** [comment]
Yes it's planned, I just have to find some good way to do it. I need to store the data in some good way and then also find a good way to display it.
